core/templates/dev/head/css/oppia.css

```diff
@@ -988,14 +988,37 @@ textarea {
 }
 
 .oppia-library-group-header-container {
-  max-width: calc(100% - 180px);
+  width: calc(100% - 143px);
```

Comment:
Is it possible to add a fallback for older browsers? http://html5please.com/#calc